release_notes/44-release-notes.adoc

```diff
@@ -0,0 +1,560 @@
+* xref:../release_notes/44-release-notes.adoc#migration-to-stock-red-hat-openshift-sccs-during-manual-upgrade-by-using-roxctl-cli_release-notes-44[Migration to stock {osp} SCCs during manual upgrade by using roxctl CLI]
+* xref:../release_notes/44-release-notes.adoc#cluster-discovery-by-using-cloud-source-integrations_release-notes-44[Cluster discovery by using cloud source integrations]
+* xref:../release_notes/44-release-notes.adoc#short-lived-api-tokens-for-central_release-notes-44[Short-lived API tokens for Central]
+
+Policy::	
+* xref:../release_notes/44-release-notes.adoc#enhanced-roxctl-deployment-check-command_release-notes-44[Enhanced roxctl deployment check command]
+* xref:../release_notes/44-release-notes.adoc#authentication-of-aws-and-gcp-integrations-by-using-short-lived-tokens_release-notes-44[Authentication of AWS and GCP integrations by using short-lived tokens (Technology Preview)]
+
+Vulnerability Management::
+* xref:../release_notes/44-release-notes.adoc#vulnerability-scanner-v4-that-utilizes-upstream-claircore_release-notes-44[Vulnerability Scanner V4 that utilizes upstream ClairCore]
+* xref:../release_notes/44-release-notes.adoc#filter-workload-cves-by-using-component-and-component-source_release-notes-44[Filter workload CVEs by using component and component source]
+
+[id="new-features_{context}"]
+== New features
+
+This release adds improvements related to the following components and concepts:
+
+[id="vulnerability-scanner-v4-that-utilizes-upstream-claircore_{context}"]
+=== Vulnerability Scanner V4 that utilizes upstream ClairCore 
+
+In {product-title-short} 4.4, the new vulnerability scanner, Scanner V4, integrates the best features of the existing StackRox Scanner and the link:https://quay.github.io/clair/whatis.html[upstream Clair V4 Scanner] that ships with Red Hat Quay, providing enhanced functionality.
+
+The following are the key highlights of the new Scanner V4:
+
+* *Consistent and accurate scanning*: Reliable vulnerability scan results across the entire Red Hat product ecosystem, Red Hat {product-title-short}, and Red Hat Quay.
+* *Expanded language and operating system support*: Expanded support for Golang in language vulnerability scanning and inclusion of Oracle Linux, SUSE Linux Enterprise, and Photon OS in operating system scanning.
+* *Comprehensive vulnerability database source*: Adoption of `OSV.dev` as the vulnerability database source for all supported programming language packages.
+
+[NOTE]
+====
+* In {product-title-short} 4.4, both the StackRox Scanner and the new Vulnerability Scanner V4 are available for scanning workloads, and the existing StackRox Scanner reports node and platform vulnerabilities.
+* {product-title-short} upgrades and new installations use the StackRox Scanner by default.
+* You now have the option to choose the new Vulnerability Scanner V4 instead of default StackRox Scanner, which offers additional benefits and an extended scope.
```

Comment:
This is not true - please confirm @RTann 

```suggestion
* You now have the option to enable the new Vulnerability Scanner V4 in addition to the default StackRox Scanner, which offers additional benefits and an extended scope.
```

Reply:
The phrasing you're suggesting sounds right. Maybe we should make it clear Scanner V4 is only for image scanning at this time while StackRox Scanner is still needed for node and platform scanning